fontforge/python.c

```diff
@@ -14066,6 +14066,150 @@ return (NULL);
 return( Py_BuildValue("d",val));
 }
 
+static PyObject *PyFF_Font_get_style_set_names(PyFF_Font *self, void *UNUSED(closure)) {
+    int cnt, i;
+    SplineFont *sf = self->fv->sf;
+    struct otffeatname *fn;
+    struct otfname *on;
+    PyObject *ss_names_tuple, *ss_name_spec;
+    char *lang_str, tag_str[5];
+
+    if ( CheckIfFontClosed(self) )
+        return( NULL );
+
+    for ( cnt=0, fn=sf->feat_names; fn!=NULL; fn=fn->next )
```

Comment:
You don't need to change this, but I think it's generally clearer to initialize variables used this way (accessed after the loop ends) before the loop rather than in it.